### PR TITLE
Add WAS table schema

### PR DESCRIPTION
**Pull request by cduhn17**
#  #

## 🗣 Description ##

Adding table to database schema: `was_tracker_customerdata`

## 💭 Motivation and context ##

The WAS team will use P&E database instance to house the data for reports that are run by the WAS team. Additionally, this table will be available via P&E UI and REST API endpoint. 

## ✅ Pre-approval checklist ##

- [x] The table schema has been added to data_schema.sql
- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: src/pe_reports/data/data_schema.sql [hunk 1]
```diff
@@ -558,6 +558,35 @@ CREATE TABLE public.unique_software (
...
+    next_scheduled           text                            not null,
+    last_scanned             text                            not null,
+    frequency                text                            not null,
+    comments_notes           text                            not null,
+    was_report_poc           text                            not null,
+    was_report_email         text                            not null,
+    onboarding_date          text                            not null,
+    no_of_web_apps           integer                         not null,
+    no_web_apps_last_updated text,
+    elections                text
```
Comment (dav3r) [inline src/pe_reports/data/data_schema.sql]: Can these fields be alphabetized or does their order matter here?  Alphabetical ordering of long lists like this is always preferable for readability, as long as the order does not matter.

Comment (dav3r) [inline src/pe_reports/data/data_schema.sql]: Looks like you are using different spacing and capitalization (e.g. `not null` vs. `NOT NULL`) than the rest of this file.  These things should be consistent throughout the file.

**Review by dav3r — COMMENTED** [review]
I left a couple of comments for your consideration.

More generally, I just wanted to confirm that it makes sense to include this WAS data in the P&E database.  If you are ok tying their data to yours (and all of the access and maintenance ramifications that come with that), then it's fine.  I am only making sure that you have thought about how this might affect things a year or two (or longer!) down the road.

File: src/pe_reports/data/data_schema.sql (hunk 1 above)
Comment (cduhn17) [reply src/pe_reports/data/data_schema.sql]: @dav3r , 
The WAS data isn't really mixing with our data at any level at this point and no real talk of integration with their data. We are just providing a space for their data and API endpoints for their team members interact with their data. I left the columns in the order that they had them so I dont change their process of writing to their flat files as they have code in place to write the flat file in this format. I do agree with you the columns s/b in alphabetical order, just doesn't fit the current process.

Comment (cduhn17) [reply src/pe_reports/data/data_schema.sql]: @dav3r , 
I have updated to capitalization per commit [c8befc4](https://github.com/cisagov/pe-reports/pull/427/commits/c8befc498db0e2d859499ea0a636994f534b8e3d)

Comment (dav3r) [reply src/pe_reports/data/data_schema.sql]: If you are just giving them a place to put their data and there are no plans to integrate it with P&E data, is there any reason why they are not getting their own database, as opposed to adding a (essentially unrelated) WAS table in the P&E database?

Comment (dav3r) [reply src/pe_reports/data/data_schema.sql]: What about the spacing differences here vs. the rest of the file?

File: setup.py [hunk 2]
```diff
@@ -122,13 +122,13 @@ def get_version(version_file):
...
         "pymongo == 4.0.1",
-        "pymupdf == 1.19.0",
+        "pymupdf",
         "python-dateutil >= 2.7.3",
         "pytest-cov",
         "python-pptx == 0.6.21",
         "pytz",
         "pyyaml == 6.0",
-        "reportlab == 3.6.6",
+        "reportlab",
```
Comment (dav3r) [inline setup.py]: The PR description should mention that you are removing these version pins and why you are doing so.

File: src/pe_reports/data/data_schema.sql (hunk 1 above)
Comment (cduhn17) [reply src/pe_reports/data/data_schema.sql]: I should rephrase that, our data isn't twisted with theirs, but we will be pulling the data to report on the data in our 
monthly reports.

File: setup.py (hunk 2 above)
Comment (cduhn17) [reply setup.py]: I have reverted these changes. It originally looked as though these pinned packages were causing problems.

Comment (dav3r) [reply setup.py]: Reverted in https://github.com/cisagov/pe-reports/pull/427/commits/1c8b9b49bf1d187d28773c186d4dd4c91aa59b87.

File: src/pe_reports/data/data_schema.sql (hunk 1 above)
Comment (dav3r) [reply src/pe_reports/data/data_schema.sql]: Got it- so it sounds like there is a need to have WAS data in the P&E database.  Thanks for clarifying!

**Review by aloftus23 — APPROVED** [review]
Looks good

File: src/pe_reports/data/data_schema.sql (hunk 1 above)
Comment (cduhn17) [reply src/pe_reports/data/data_schema.sql]: @dav3r , 
I have addressed your comment in commit  [1316325](https://github.com/cisagov/pe-reports/pull/427/commits/13163255da333acc357b99c8bc48169fb228c316)

File: src/pe_reports/data/data_schema.sql [hunk 3]
```diff
@@ -558,6 +558,33 @@ CREATE TABLE public.unique_software (
     software_name text NOT NULL
 );
 
+--
+--Name: was_tracker_customerdata
```
Comment (dav3r) [inline src/pe_reports/data/data_schema.sql]: This should match the other tables:
```suggestion
-- Name: was_tracker_customerdata; Type: TABLE; Schema: public; Owner: pe
```

**Review by dav3r — COMMENTED** [review]
The PR description also mentions `vw_cidrs`, but I don't see anything in this PR about a table or schema with that name.

**Comment by cduhn17** [comment]
@dav3r , 

I removed the reference to `vw_cidrs` in the PR description.

**Review by dav3r — APPROVED** [review]
LGTM 👍

**Pull request merged**